### ISSUE TITLE
Natives incompatibility with MacOS

### ISSUE THREAD

**Issue by ChloeDawn** [SKCraft/Launcher#383]
**Launcher version:** 3a7c2507366292921a959aec59b31128213704a7
**MacOS version:** 11.1 (Big Sur)

The launcher cannot provide the correct natives on MacOS because it looks for `osx` natives when only `macos` natives exist
```
java.lang.NullPointerException
    at com.skcraft.launcher.model.minecraft.Library.getNativeString(Library.java:70)
    at com.skcraft.launcher.model.minecraft.Library.getArtifact(Library.java:94)
    at com.skcraft.launcher.update.BaseUpdater.installLibraries(BaseUpdater.java:225)
    at com.skcraft.launcher.update.Updater.update(Updater.java:194)
    at com.skcraft.launcher.update.Updater.call(Updater.java:97)
    at com.skcraft.launcher.update.Updater.call(Updater.java:40)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at java.lang.Thread.run(Thread.java:748)
```
~~The following natives lookup needs to use `"macos"` if `"osx"` is null (although I think `osx` may be entirely redundant now? Unsure.)~~
https://github.com/SKCraft/Launcher/blob/f53ee9b6faf5af222b38da3c1d89b2bbedf4f785/launcher/src/main/java/com/skcraft/launcher/model/minecraft/Library.java#L64

**Comment by ChloeDawn** [comment]
On further inspection I misread the JSON. It seems there is still `osx` in `natives`, but the classifier is `natives-macos`

**Comment by hedgehog1029** [comment]
Introduced in ca08bbd, fixed in d0d8ad79eee873c8c6c2dd40fb16ca7b7f7e521a
